ansible/files/bootstrap/telemetry.sh

```diff
@@ -62,6 +26,14 @@ while [ $# -gt 0 ]; do
             shift
             TIMEOUT=$1
             ;;
+        --email)
+            shift
+            EMAIL=$1
+            ;;
+        --efs-dns)
+            shift
+            EFS_DNS=$1
```

Comment:
@fbertsch Is this something that we need to add in https://github.com/mozilla/telemetry-analysis-service/issues/217 as well?